source/index.js

```diff
@@ -23,6 +22,7 @@ const npm = require('./npm/util');
 const releaseTaskHelper = require('./release-task-helper');
 const util = require('./util');
 const git = require('./git-util');
+const githubChecks = require('./github-checks');
```

Comment:
I'd rename this to `verifyGithubChecksPassed` (or something similar).

Reply:
What do you think about something more generic such as `verifyCommitChecks` or `verifyCommitStatus`? I already have one eye on [GitLab’s commit status API](https://docs.gitlab.com/ee/api/commits.html#commit-status) so it might make sense not to tie the terminology to GitHub (we could then also rename the `--no-status-checks` command-line option).

Reply:
`verifyCommitStatusChecks`

Reply:
Renamed in 4cfcb43f7b79da53b9d7ea58eabe98819ae46b3d. Let me know if you’d also like to rename the command-line option and config flag but I think they both benefit from a more succinct name.